### PR TITLE
Create Footer

### LINKED ISSUE #156: The P40000000 Footer
## Description

image

No details bc I'm lazy

### PR DESCRIPTION
**Pull request by amit-sawhney**
## Summary

Creates the footer in a somewhat responsive way. 
Closes #156 

## Changes

- Add social icon components to assets folder
- Add footer logic

## Screenshots

Screen Shot 2023-04-27 at 10 21 29 PM

### CODE REVIEW

**Comment by ajain1921** [comment]
hmm im not sure react is the best choice for this PR. would u mind rewriting in rust?

**Comment by amit-sawhney** [comment]
I apologize you are right.

**Comment by zorazrr** [comment]
...

**Comment by amit-sawhney** [comment]
@ajain1921 I have addressed your changes

**Comment by ajain1921** [comment]
> @ajain1921 I have addressed your changes

LOL WHAT??? HOW DID U EVEN MAKE THIS?? WHY DID THIS SUCCESSFULLY DEPLOY? SO MANY QUESTIONS...

**Review by zorazrr — CHANGES_REQUESTED** [review]
🐐

File: src/components/Icon/FacebookIcon.tsx
Comment (zorazrr) [inline src/components/Icon/FacebookIcon.tsx:1]: Also is it me or is the Facebook icon smaller than the other ones.

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+                  </span>
+                ),
+              },
+            ]}
+          />
+          <LinkSubsection
+            title="Follow SECS"
+            links={[
+              {
+                href: 'https://www.instagram.com/secsillinois/',
```
Comment (zorazrr) [inline src/components/Footer/Footer.tsx:111]: This link does not work?? Should it be https://www.instagram.com/secsuiuc/?

File: src/pages/home/index.tsx [hunk 2]
```diff
@@ -89,6 +94,8 @@ const Home: FC<HomeProps> = ({
         </div>
 
         <ToTopButton />
+
+        <Footer />
```
[the remark this answers is not in the document]
Comment (amit-sawhney) [reply src/pages/home/index.tsx:98]: wait there isn't just one home page for all of your main pages 😭

File: src/components/Icon/FacebookIcon.tsx
Comment (amit-sawhney) [reply src/components/Icon/FacebookIcon.tsx:1]: it is, but the icon itself is the same size, so wasn't sure, let me see if i can fix

File: src/components/Footer/Footer.tsx (hunk 1 above)
Comment (amit-sawhney) [reply src/components/Footer/Footer.tsx:111]: OH MY BAD I FORGOT TO ACTUALLY GO FIX THE LINKS

**Review by zorazrr — APPROVED** [review]

**Pull request merged**